test/test__helper.rb

```diff
@@ -33,12 +33,30 @@
   SimpleCov.formatter = SimpleCov::Formatter::Codecov
 end
 
+class Object
+  def truthy?
```

Comment:
@mgrassotti without usage reference, hard to guess what it does by just looking at the name. I propose `#truthy_env_var?`.

Reply:
@ledestin thanks, I see your point. But the method can be applied also to general `Nil` and `String` objects, stating if they are "truthy". Do you think the method name should include where we are using it anyway?

Reply:
@mgrassotti well, it can be applied, but it doesn't say why. There is already a definition of truthiness in Ruby, and you add another one. People will ask "Why? What is the difference?".

On second thought, a call should be `what#action`, but in our case, action also contains what. So maybe do a helper method `truthy_env_var?('VAR_NAME')`?

Reply:
@ledestin ok. As this becomes a helper, I included it in the `Test` class, avoiding to extend `Object`